src/main/java/org/datadog/jmxfetch/HttpClient.java

```diff
@@ -21,6 +22,11 @@ public class HttpClient {
     private int port;
 
     private static final String USER_AGENT = "Datadog/JMXFetch";
+    // Per javadocs, this is the only version that all compliant JVMs are required to support
+    // I found 'TLS' was the appropriate protocol (will use the latest support TLSv? version)
+    // rather than specifically 'TLSv1' as the docs recommend
+    // https://docs.oracle.com/javase/7/docs/api/javax/net/ssl/SSLContext.html
+    private static final String minRequiredSSLProtocol = "TLS";
```

Comment:
Would it make sense to make it configurable? Something like:

```suggestion
    private static final String minRequiredSSLProtocol = System.getProperty("jmxfetch.min_tls_version", "TLS");
```